sdcclient/_scanning.py

```diff
@@ -291,6 +291,29 @@ def check_image_evaluation(self, image, show_history=False, detail=False, tag=No
 
         return [True, res.json()]
 
+    def get_pdf_report(self, image, pdfpath, tag=None, date=None):
+        itype, _, image_digest = self._discover_inputimage(image)
```

Comment:
There are few variable names that are a little confusing compared to other ones. Would it be possible to review the naming of things like `itype`, `thetag` and `pdfpath` to make the a little more self-explanatory and using the snake case convention used in (most of) the library? Thanks!